src/lib/Scenes/Collection/Components/__tests__/FeaturedArtists-tests.tsx

```diff
@@ -87,11 +90,11 @@ const CollectionFixture: FeaturedArtists_collection = {
 describe("FeaturedArtists", () => {
   const render = (collection: FeaturedArtists_collection) =>
     renderRelayTree({
-      Component: ({ marketingCollection }) => (
+      Component: mockTracking(({ marketingCollection }) => (
```

Comment:
Using the `mockTracking` pattern described [here](https://github.com/artsy/emission/blob/724c929428f57000c31d4d14444a9db8deb39203/src/lib/tests/mockTracking.ts).

This seems a good fit for this use case: relay container around a a decorated class component with has a decorated tracking method — whew!